src/arkode/arkode_butcher_dirk.c

```diff
@@ -600,6 +609,431 @@ ARKodeButcherTable ARKodeButcherTable_LoadDIRK(ARKODE_DIRKTableID imethod)
+    B->b[1] = (RCONST(2398.0)+RCONST(1205.0)*SUNRsqrt(RCONST(2.0)))/RCONST(2835.0)/(RCONST(4.0)+RCONST(3.0)*SUNRsqrt(RCONST(2.0)));
+    B->b[2] = -RCONST(2374.0)*(RCONST(1.0)+RCONST(2.0)*SUNRsqrt(RCONST(2.0)))/RCONST(2835.0)/(RCONST(5.0)+RCONST(3.0)*SUNRsqrt(RCONST(2.0)));
+    B->b[3] = RCONST(5827.0)/RCONST(7560.0);
+    B->b[4] = RCONST(9.0)/RCONST(40.0);
+
+    B->d[0] = RCONST(4555948517383.0)/RCONST(24713416420891.0);
+    B->d[1] = RCONST(4555948517383.0)/RCONST(24713416420891.0);
+    B->d[2] = -RCONST(7107561914881.0)/RCONST(25547637784726.0);
+    B->d[3] = RCONST(30698249.0)/RCONST(44052120.0);
+    B->d[4] = RCONST(49563.0)/RCONST(233080.0);
+
+    B->A[1][0] = RCONST(9.0)/RCONST(40.0);
+    B->A[1][1] = RCONST(9.0)/RCONST(40.0);
+    B->A[2][0] = RCONST(9.0)*(RCONST(1.0)+SUNRsqrt(RCONST(2.0)))/RCONST(80.0);
+    B->A[2][1] = RCONST(9.0)*(RCONST(1.0)+SUNRsqrt(RCONST(2.0)))/RCONST(80.0);
+    B->A[2][2] = RCONST(9.0)/RCONST(40.0);
+    B->A[3][0] = (RCONST(22.0)+RCONST(15.0)*SUNRsqrt(RCONST(2.0)))/RCONST(80.0)/(RCONST(1.0)+SUNRsqrt(RCONST(2.0)));
+    B->A[3][1] = (RCONST(22.0)+RCONST(15.0)*SUNRsqrt(RCONST(2.0)))/RCONST(80.0)/(RCONST(1.0)+SUNRsqrt(RCONST(2.0)));
+    B->A[3][2] = -RCONST(7.0)/RCONST(40.0)/(RCONST(1.0)+SUNRsqrt(RCONST(2.0)));
+    B->A[3][3] = RCONST(9.0)/RCONST(40.0);
+    B->A[4][0] = B->b[0];
+    B->A[4][1] = B->b[1];
+    B->A[4][2] = B->b[2];
+    B->A[4][3] = B->b[3];
+    B->A[4][4] = B->b[4];
+    break;
+
+  case(ARKODE_ESDIRK32I5L2SA_5_2_3):   /* ESDIRK3(2I)5L[2]SA (A,L stable) */
+    B = ARKodeButcherTable_Alloc(5, SUNTRUE);
+    B->q = 3;
+    B->p = 2;
+
+    B->c[1] = RCONST(9.0)/RCONST(20.0);
+    B->c[2] = RCONST(4.0)/RCONST(5.0);
+    B->c[3] = RCONST(1.0);
+    B->c[4] = RCONST(1.0);
+
+    B->b[0] = RCONST(7415.0)/RCONST(34776.0);
+    B->b[1] = RCONST(9920.0)/RCONST(30429.0);
+    B->b[2] = RCONST(4845.0)/RCONST(9016.0);
+    B->b[3] = -RCONST(5827.0)/RCONST(19320.0);
+    B->b[4] = RCONST(9.0)/RCONST(40.0);
+
+    B->d[0] = RCONST(23705.0)/RCONST(104328.0);
+    B->d[1] = RCONST(29720.0)/RCONST(91287.0);
+    B->d[2] = RCONST(4225.0)/RCONST(9016.0);
+    B->d[3] = -RCONST(69304987.0)/RCONST(337732920.0);
+    B->d[4] = RCONST(42843.0)/RCONST(233080.0);
+
+    B->A[1][0] = RCONST(9.0)/RCONST(40.0);
+    B->A[1][1] = RCONST(9.0)/RCONST(40.0);
+    B->A[2][0] = RCONST(19.0)/RCONST(72.0);
+    B->A[2][1] = RCONST(14.0)/RCONST(45.0);
+    B->A[2][2] = RCONST(9.0)/RCONST(40.0);
+    B->A[3][0] = RCONST(3337.0)/RCONST(11520.0);
+    B->A[3][1] = RCONST(233.0)/RCONST(720.0);
+    B->A[3][2] = RCONST(207.0)/RCONST(1280.0);
+    B->A[3][3] = RCONST(9.0)/RCONST(40.0);
+    B->A[4][0] = B->b[0];
+    B->A[4][1] = B->b[1];
+    B->A[4][2] = B->b[2];
+    B->A[4][3] = B->b[3];
+    B->A[4][4] = B->b[4];
+    break;
+
+  case(ARKODE_ESDIRK436L2SA_6_3_4):   /* ESDIRK4(3)6L[2]SA (A,L stable) */
```

Comment:
Main coefficients are exact but embedded are correct to ~26 digits

Reply:
Thanks for verifying.